src/htmlrenderer.cpp

```diff
@@ -683,7 +683,9 @@ void HtmlRenderer::render(std::istream& input,
 						tables.size() ? 0
 						: indent_level);
 				}
-				list_elements_stack.pop_back();
+				if (!list_elements_stack.empty()) {
+					list_elements_stack.pop_back();
+				}
```

Comment:
It looks like there is some kind of rendering issue in Github's diff viewer. The indentation looks quite different from the indentation I'm seeing locally in vim (specifically the indentation of the changed code compared with the switch case indentation)
It looks like only tabs are used for indentation (as expected), so I'm not sure what is going on

![image](https://user-images.githubusercontent.com/4629607/155013690-31966a7a-a800-4fac-a232-e6e0545528ab.png)
![image](https://user-images.githubusercontent.com/4629607/155013820-e1ea5b8a-b528-4697-9084-697fd1e21680.png)

Reply:
Yeah, looks like a GitHub glitch. The file is fine, so this doesn't matter :) Thanks for the heads-up though!